app/models/manageiq/providers/google/cloud_manager/provision/configuration.rb

```diff
@@ -0,0 +1,2 @@
+module ManageIQ::Providers::Google::CloudManager::Provision::Configuration
+end
```

Comment:
If you do not need to add or override anything here this file is not needed.  You can also remove the `include_concern 'Configuration'` in provision.rb.